test/multi_namespace/src/foo.gleam

```diff
@@ -0,0 +1,3 @@
+pub fn main() {
```

Comment:
Rename this file to something else please, "foo" is a morbid world-war-2 reference.